lib/theme_check/node.rb

```diff
@@ -132,12 +162,81 @@ def position
       )
     end
 
+    def start_token
+      return "" if inside_liquid_tag?
+      output = ""
+      output += "{{" if variable?
+      output += "{%" if tag?
+      output += "-" if whitespace_trimmed_start?
```

Comment:
I don't think whitespace trimming works in variables? So perhaps move this inside the `if` I'm suggesting at the top?

Reply:
It's supposed to!
https://shopify.github.io/liquid/basics/whitespace/